commands/export_test.go

```diff
@@ -9,6 +9,8 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/10gen/realm-cli/utils/telemetry"
+
```

Comment:
no newline here (and the existing import group could use a little driveby fixin too)